tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -37,15 +37,7 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
     // random inputs.
     val cudfInvalidPatterns = Seq(
       "a*+",
-      "\t+|a",
```

Comment:
Many of these patterns now compile in cuDF but don't necessarily work, so we still reject them in the transpiler. 

TODO: modify this test rather than just delete these patterns